pylarky/tests/test_lib.py

```diff
@@ -68,7 +68,7 @@ def process(input_message):
         == '{"credit_card":"411111111111111111","cvv":"043","expiration_date":"03/43","signature":"12b03226a6d8be9c6e8cd5e55dc6c7920caaa39df14aab92d5e3ea9340d1c8a4d3d0b8e4314f1f6ef131ba4bf1ceb9186ab87c801af0d5c95b1befb8cedae2b9"}'
     )
     assert modified_request.headers == {
-        "Content-Type": "application/json",
+        "Content-type": "application/json",
```

Comment:
So this matches the python behavior now right ?

Reply:
yes, this is the official python urllib behavior